### PR TITLE
Audiolink extras

### PR DESCRIPTION
**Pull request by float3**

### CODE REVIEW

**Comment by float3** [comment]
disable CRT is misnamed, TODO: rename it

File: Packages/com.llealloo.audiolink.extras/Runtime/Scripts/AudioLinkDisableCRT.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        public AudioLink audioLink;
+        private RenderTexture originalRenderTexture;
+        private int _AudioTexture;
+
+        void Start()
+        {
+            _AudioTexture = VRCShader.PropertyToID("_AudioTexture");
+        }
+
+        public void _DisableCRT()
```
Comment (pema99) [inline Packages/com.llealloo.audiolink.extras/Runtime/Scripts/AudioLinkDisableCRT.cs:21]: whats with the underscores? Also is it intentional that these are never called?

**Comment by pema99** [comment]
you gunning to put this in 0.3.2?

**Comment by pema99** [comment]
oh, actually wait, it doesn´t matter since its in the extras package

**Comment by float3** [comment]
> you gunning to put this in 0.3.2?

gonna wait for Nestors PR before I pick this one up again

**Comment by float3** [comment]
made obsolete by #238

**Pull request closed without merging**